src/main.rs

```diff
@@ -48,11 +66,37 @@ fn main() {
     let num_segments = state.full_n_segments().expect("failed to get number of segments");
 
     println!("{}", num_segments);
+    
+    let mut srt_sequences: Vec<String> = Vec::new();
+    let mut timestamped_lines: Vec<String> = Vec::new();
 
     for i in 0..num_segments {
         let segment = state.full_get_segment_text(i).expect("failed to get segment");
         let start_timestamp = state.full_get_segment_t0(i).expect("failed to get segment start timestamp");
         let end_timestamp = state.full_get_segment_t1(i).expect("failed to get segment end timestamp");
-        println!("[{} - {}]: {}", start_timestamp, end_timestamp, segment);
+
+        let srt_start_timestamp = milliseconds_to_srt_time_string(start_timestamp);
+        let srt_end_timestamp = milliseconds_to_srt_time_string(end_timestamp);
+        let srt_formatted: String = format!("{}\n{} --> {}\n{}\n\n", i+1, srt_start_timestamp, srt_end_timestamp, segment);
```

Comment:
```suggestion
        let srt_formatted: String = format!("{}\n{srt_start_timestamp} --> {srt_end_timestamp}\n{segment}\n\n", i+1);
```

Is something like this easier?